src/app/home/home.component.ts

```diff
@@ -43,9 +51,31 @@ export class HomeComponent implements OnInit {
         this.products = data
         if (this.products.count) { this.isProducts = true } else { this.isProducts = false }
       })
+      this.store.select(getTaxonomies)
+      .subscribe(data => {
+        this.fillterList = data; 
+        console.log(this.fillterList);
```

Comment:
Remove this console log.

Reply:
Work in progress on that filter mobile menu